src/Quasar/Error.purs

```diff
@@ -24,22 +24,26 @@ import Control.Monad.Eff.Exception (Error, error, message)
 data QError
   = NotFound
   | Forbidden
+  | PaymentRequired
   | Error Error
 
 instance showQError ∷ Show QError where
   show NotFound = "NotFound"
   show Forbidden = "Forbidden"
+  show PaymentRequired = "PaymentRequired"
   show (Error err) = "(Error " <> show err <> ")"
 
 printQError ∷ QError → String
-printQError NotFound = "Resource not found"
-printQError Forbidden = "Resource is unavailable, authorization is required first"
-printQError (Error err) = message err
+printQError = case _ of
+  NotFound → "Resource not found"
+  Forbidden → "Resource is unavailable, authorization is required first"
+  PaymentRequired → "Resource is unavailable, payment is required to use this feature"
```

Comment:
@jdegoes have you got any thoughts on how this message could be phrased better?

I think we'll be handling it manually in SlamData so can use a better message there anyway, but it's probably worth having something decent here too.